cmd/state-offline-installer/runInstall.go

```diff
@@ -149,13 +145,9 @@ func runInstall(out output.Outputer, params *Params) error {
 	out.Print(fmt.Sprintf("Stage 3 of 3 Finished: Installing artifacts from: %s", artifactsPath))
 
 	// Copy license file
-	contents, err := assets.ReadFileBytes(TOSPath)
-	if err != nil {
-		return errs.Wrap(err, "Error reading file bytes for TOS")
-	}
-	err = fileutils.WriteFile(licenseFilePath, contents)
+	err = fileutils.CopyFile(licenseFileAssetPath, licenseFilePath)
```

Comment:
@shaunmlowry I feel like if we want to install runtimes with a license file then the license should be part of the runtime, rather than something we manually account for. Thoughts?

Reply:
Maybe we'd get there at some point, but trying to get another artifact (the license) through the build planner into the final artifact risks delaying build plan support for installers even further

Reply:
I'm ok with it. Most of the burden will be on the thing that produces the offline installer payload rather than the installer itself.